addons/sale_timesheet/static/tests/tours/sale_timesheet_tour.js

```diff
@@ -0,0 +1,262 @@
+odoo.define('sale_timesheet.tour', function (require) {
+"use strict";
+
+const tour = require('web_tour.tour');
+
+tour.register('sale_timesheet_tour', {
+    test: true,
+    url: '/web',
+}, [...tour.stepUtils.goToAppSteps("sale.sale_menu_root", 'Go to the Sales App'),
+{
+    trigger: 'button.o_list_button_add',
+    content: 'Click on CREATE button to create a quotation with service products.',
+    run: 'click',
+}, {
+    trigger: 'div[name="partner_id"]',
+    content: 'Add the customer for this quotation (e.g. Brandon Freeman)',
+    run: function (actions) {
+        actions.text('Brandon Freeman', this.$anchor.find('input'));
+    },
+}, {
+    trigger: 'ul.o_partner_autocomplete_dropdown > li:first-child > a',
```

Comment:
Hello @xavierbol ,
I am currently in the process of re-writing the partner autocomplete widget to Owl and I discovered that this test was interacting with it.
I think it'd better to avoid interacting with it as the partner autocomplete module isn't part of your dependencies.
For now, I simply adapted the tour so it will still work as before (see #101098).

Reply:
Hello @lobaudoux , oh indeed my bad, thanks for the fix. 🙂